lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -355,7 +370,7 @@ def _parse_account_key(self, key):
 
         pub_hex, pub_exp = re.search(
             r"modulus:\n\s+00:([a-f0-9\:\s]+?)\npublicExponent: ([0-9]+)",
-            out.decode('utf8'), re.MULTILINE | re.DOTALL).groups()
+            to_text(out), re.MULTILINE | re.DOTALL).groups()
```

Comment:
Unless simply constructing a message for the user, use to_text, to_bytes, etc with errors='surrogate_or_strict' like this:

``` python
to_text(out, errors='surrogate_or_strict'), re.MULTILINE | re.DOTALL).groups()
```

Reply:
Ok, have added the error handler argument.